node/hubsConfig.js

```diff
@@ -8,7 +8,22 @@ var hubs = [
         'id': "winkHub",
         'name': 'Wink',
         'translator': 'opent2t-translator-com-wink-hub'
-    }
+    },
+    { 
+        'id': "insteonHub",
+        'name': 'Insteon',
+        'translator': 'opent2t-translator-com-insteon-hub'
+    },
+    { 
+        'id': "nestHub",
+        'name': 'Nest',
+        'translator': 'opent2t-translator-com-nest-hub'
+    },
+    { 
+        'id': "smartthingsHub",
+        'name': 'SmartThings',
+        'translator': 'opent2t-translator-com-smartthings-hub'
+    },
```

Comment:
I believe that's allowed in .json syntax but removed none-the-less. TS linters actually want the extra comma !